packages/grid/src/vaadin-grid-drag-and-drop-mixin.js

```diff
@@ -291,6 +309,26 @@ export const DragAndDropMixin = (superClass) =>
       }
     }
 
+    /** @private */
+    __onGridDragStart(e) {
```

Comment:
Would suggest renaming this to `__onDocumentDragStart` or such to avoid confusing it with the `grid-dragstart` event

Reply:
Done.